src/commands/MigrationGenerateCommand.ts

```diff
@@ -7,6 +7,7 @@ import {camelCase} from "../util/StringUtils";
 import * as yargs from "yargs";
 import {AuroraDataApiDriver} from "../driver/aurora-data-api/AuroraDataApiDriver";
 import chalk from "chalk";
+import { format } from "@sqltools/formatter/lib/sqlFormatter";
```

Comment:
is it a safe import? Don't they have it exported on `@sqltools/formatter` ?

Reply:
Unfortunately the TS compiler's unhappy with that (`Could not find a declaration file for module '@sqltools/formatter'`).. The declaration file is in there but I think it's unable to pick it up from the top level folder.

Require statement works ok, if you prefer that? `const { format } = require("@sqltools/formatter");`

I'm a bit newb-ish on how the packaging works though - here's the contents of `node_modules/@sqltools/formatter` if you can spot anything in that structure:

![image](https://user-images.githubusercontent.com/26905074/92970940-72a62c80-f477-11ea-8a26-5114c48596c1.png)

Reply:
okay, lets merge it as it is... but I bet if there is something wrong with typing on `@sqltools/formatter` they will release a new version with change and it can break our stuff... so I would probably fix this package version for now.